test/object-store/realm.cpp

```diff
@@ -2367,12 +2367,6 @@ TEST_CASE("SharedRealm: schema updating from external changes") {
         WriteTransaction wt(db);
         auto& table = *wt.get_table("class_object");
 
-        SECTION("removing a property") {
```

Comment:
This test does no longer apply for additive schemas, since in case of external changes, and in case of removed properties, all the missing properties will be appended to the end of the `persisting_properties` array.

Reply:
I think it is better to keep this test, but check that there is no exception thrown and that the column still exists.

Reply:
OK, but I think an exception is going to be thrown. I think `invalidate` is doing something on the primary key... but let's keep the test either way. Just with the opposite logic.

Reply:
This test is still applicable and does not need any changes. Nothing in this PR makes it no longer a problem if an external process removes a column that we're using.

Reply:
This should now be solved.